students/jungmin/westagram/users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path, include
```

Comment:
```suggestion
from django.urls import path
```

`westagram/urls.py` 가 아닌 `users/urls.py`에서는 Include가 쓰이지 않고 있습니다.
필요하지 않으므로 지워주세요~